extensions/amp-next-page/1.0/test/test-amp-next-page.js

```diff
@@ -479,6 +483,146 @@ describes.realWin(
       });
     });
 
+    describe('default separators', () => {
+      let element;
+      let service;
+
+      beforeEach(async () => {
+        element = await getAMPNextPage({
+          inlineConfig: VALID_CONFIG,
+        });
+
+        await element.build();
+        await element.layoutCallback();
+
+        service = Services.nextPageServiceForDoc(doc);
+        env.sandbox.stub(service, 'getViewportsAway_').returns(2);
+      });
+
+      afterEach(async () => {
+        element.parentNode.removeChild(element);
+      });
+
+      it('adds a default separator to for the host page', async () => {
+        await fetchDocuments(service, MOCK_NEXT_PAGE_WITH_RECOMMENDATIONS);
+
+        expect(service.pages_[1].container.firstElementChild).to.have.class(
+          'amp-next-page-default-separator'
+        );
+      });
+
+      it('adds a default separator between embedded pages', async () => {
+        await fetchDocuments(service, MOCK_NEXT_PAGE, 2);
+
+        expect(service.pages_[2].container.firstElementChild).to.have.class(
+          'amp-next-page-default-separator'
+        );
+      });
+    });
+
+    describe('custom and templated separators', () => {
+      let element;
+      let service;
+
+      afterEach(async () => {
+        element.parentNode.removeChild(element);
+      });
+
+      it('renders a custom separator correctly', async () => {
+        const separator = doc.createElement('div');
```

Comment:
Both of the separator tests have the same setup logic of calling `getAMPNextPage` with the `separator` and building/laying out the element. To make this more readable, we can separate it out into a method that takes the `separator` and returns the component after `layoutCallback`.